esp-radio/src/fmt.rs

```diff
@@ -1,5 +1,6 @@
 #![macro_use]
 #![allow(unused_macros)]
+#![allow(unused)]
```

Comment:
should be enough to only have this on `Bytes` itself

Reply:
It's fine on the whole file, I don't really want to have differences between fmt.rs files, and so we shouldn't care about what is unused where.